src/seeds/seeder.module.ts

```diff
@@ -13,7 +20,9 @@ const REPOS = [SeederRepository, { provide: Users.name, useValue: Users }];
 export class SeederModule implements OnApplicationBootstrap {
   constructor(private readonly seederService: SeederService) {}
 
-  async onApplicationBootstrap(): Promise<any> {
-    return this.seederService.startSeedsUsers();
+  async onApplicationBootstrap() {
+    await this.seederService.startSeedsGroups();
+    await this.seederService.startSeedsUsers();
+    await this.seederService.startSeedsUsersGroups();
```

Comment:
You can create one public method `seed` in the `SeederService` which will execute necessary private `seedSmth` methods.

Reply:
I will fix it when I refactor seeder